src/test.rs

```diff
@@ -0,0 +1,34 @@
+use std::io::{BufRead, BufReader};
+
+use clap::ArgMatches;
+use duct::cmd;
+
+use crate::{error::AocError, util::get_day};
+
+pub async fn test(matches: &ArgMatches) -> Result<(), AocError>
+{
+    let day = get_day(matches)?;
+
+    let reader = cmd!(
+        "cargo",
+        "test",
+        "--color",
```

Comment:
Here we actually have to duplicate `--color always` because the first one gives color to the warnings and possible compile errors etc... And the other one gives color to the output from the tests. So this is intended